src/Cake.Npm/NpmAliases.cs

```diff
@@ -0,0 +1,359 @@
+namespace Cake.Npm
+{
+    using System;
+    using Core;
+    using Core.Annotations;
+    using Install;
+    using Pack;
+    using RunScript;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Provides a wrapper around Npm functionality within a Cake build script
+    /// </summary>
+    [CakeAliasCategory("Npm")]
+    [CakeNamespaceImport("Cake.Npm.Install")]
+    [CakeNamespaceImport("Cake.Npm.Pack")]
+    [CakeNamespaceImport("Cake.Npm.RunScript")]
+    public static class NpmAliases
```

Comment:
I'm looking at the merge now, so I can do that.  Also looking to update the basic readme and `usage.cake` information.